SamplesDashboard/SamplesDashboard/ClientApp/src/views/Repositories.tsx

```diff
@@ -279,6 +279,28 @@ function copyAndSort<T>(items: T[], columnKey: string, isSortedDescending?: bool
     return itemsSorted;
 }
 
+function displayAdmins(owner: any)
+{
+    if (owner != null) {
+        var user = document.createElement('div');
```

Comment:
>user [](start = 12, length = 4)

from a naming standpoint.  This is more the container, and item represents the user